StratisCore.UI/main.ts

```diff
@@ -343,8 +343,8 @@ function startDaemon() {
   
   if (instance == 1) {
 	  spawnArgs.push('-bootstrap=' + bootstrap)
-	  spawnArgs.push('-defaultwalletmnemonic', 'basic exotic crack drink left judge tourist giggle muscle unique horn body')
-	  spawnArgs.push('-poaminingkey', 'basic exotic crack drink left judge tourist giggle muscle unique horn body')
+	  spawnArgs.push('-defaultwalletmnemonic=basic exotic crack drink left judge tourist giggle muscle unique horn body')
```

Comment:
We won't need those two lines this anymore. This will now be done from the container

Reply:
Will remove